### PR TITLE
Sgr sessions pusher

### PR DESCRIPTION
**Pull request by 5v3nn**
# Pull Requests Checklist

Assignee:

- \[x\] code commented?
- \[x\] tests added / passed?
- \[x\] are tests covering all cases?
- \[x\] documentation added / updated?
- \[x\] pre-commit checks passed?
- \[x\] pipelines passed?

Reviewer:

- \[ \] are there any code smells?
- \[ \] did pipeline pass?
- \[ \] is the code well documented?
- \[ \] are the tests passing and covering all cases?

### CODE REVIEW

**Comment by 5v3nn** [comment]
# Coverage

image

File: build.gradle [hunk 1]
```diff
@@ -42,20 +42,16 @@ dependencies {
...
-    implementation 'org.springframework:spring-jdbc:6.0.6'
+    runtimeOnly 'org.postgresql:postgresql:42.6.0'
+
+    implementation 'com.pusher:pusher-http-java:1.3.3'
 
-    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.springframework.boot:spring-boot-starter-jdbc'
-    implementation 'com.google.cloud.sql:postgres-socket-factory:1.11.0'
 }
```
Comment (5v3nn) [inline build.gradle:55]: @5v3nn not ok

Comment (5v3nn) [reply build.gradle]: done

**Review by Alihan26 — APPROVED** [review]
All right all right very nice, nice coverage, nice tests!

**Pull request merged**